docs/content/references/metrics/agent.md

```diff
@@ -0,0 +1,132 @@
+| flowcontrol_error_reasons_total  | Counter | error_reason  | count(no unit)  | Number of error reasons other than unspecified  |
+| flowcontrol_reject_reasons_total | Counter | reject_reason | count (no unit) | Number of reject reasons other than unspecified |
+
+### FlowControl Labels
+
+| Name          | Example                                                                                                                                              | Description                                              |
+| ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
+| decision_type | DECISION_TYPE_ACCEPTED, DECISION_TYPE_REJECTED                                                                                                       | Whether the flow was accepted or not                     |
+| error_reason  | ERROR_NONE, ERROR_MISSING_TRAFFIC_DIRECTION, ERROR_INVALID_TRAFFIC_DIRECTION, ERROR_CONVERT_TO_MAP_STRUCT, ERROR_CONVERT_TO_REGO_AST, ERROR_CLASSIFY | Error reason for FlowControl Check response.             |
+| reject_reason | REJECT_REASON_NONE, REJECT_REASON_RATE_LIMITED, REJECT_REASON_CONCURRENCY_LIMITED                                                                    | Reason why FlowControl Check response rejected the flow. |
+
+### Distributed Cache Metrics
+
+| Name                    | Type  | Labels                                     | Unit            | Description                                                                           |
+| ----------------------- | ----- | ------------------------------------------ | --------------- | ------------------------------------------------------------------------------------- |
+| distcache_entries_total | Gauge | distcache_member_id, distcache_member_name | count (no unit) | Total number of entries in the DMap                                                   |
+| distcache_delete_hits   | Gauge | distcache_member_id, distcache_member_name | count (no unit) | Number of deletion requests resulting in an item being removed in the DMap            |
+| distcache_delete_misses | Gauge | distcache_member_id, distcache_member_name | count (no unit) | Number of deletion requests for missing keys in the DMap                              |
+| distcache_get_misses    | Gauge | distcache_member_id, distcache_member_name | count (no unit) | Number of entries that have been requested and not found in the DMap                  |
+| distcache_get_hits      | Gauge | distcache_member_id, distcache_member_name | count (no unit) | Number of entries that have been requested and found present in the DMap              |
+| distcache_evicted_total | Gauge | distcache_member_id, distcache_member_name | count (no unit) | Total number of entries removed from cache to free memory for new entries in the DMap |
```

Comment:
Labels such as instance are present on all Agent metrics. Might be a good idea to refer to metrics explorer in Prometheus

Reply:
Done.